content/docs/03-ai-sdk-core/15-tools-and-tool-calling.mdx

```diff
@@ -639,35 +639,49 @@ Create an MCP client using either:
 
 - `stdio`: Uses standard input and output streams for communication, ideal for local tool servers running on the same machine (like CLI tools or local services)
 - `SSE` (Server-Sent Events): Uses HTTP-based real-time communication, better suited for remote servers that need to send data over the network
+- Custom transport: Bring your own transport by implementing the `MCPTransport` interface
 
 ```typescript
 // Example of using MCP with stdio
-const mcpClient = await createMCPClient({
-  transport: {
-    type: 'stdio',
+const mcpClient = await experimental_createMCPClient({
+  transport: new StdioClientTransport({
     command: 'node',
     args: ['src/stdio/dist/server.js'],
-  },
+  }),
 });
 
 // Example of using MCP with SSE
-const mcpClient = await createMCPClient({
+const mcpClient = await experimental_createMCPClient({
   transport: {
     type: 'sse',
     url: 'https://my-server.com/sse',
   },
 });
+// Can also be written as:
+const mcpClient = await experimental_createMCPClient({
+  transport: new SSEMCPTransport({
```

Comment:
it already looks ugly, so wdyt about `Experimental_SSEMCPTransport`? 

Also open to ideas on how to make it look nicer - CamelCase maybe?

Reply:
Or `Sse`?